arrows/mvg/applets/tests/data/track_features_testing.conf

```diff
@@ -0,0 +1,31 @@
+# Custom configuration for testing the track-features applet
+
+# If true, all mask images will be inverted after loading.
+# This is useful if mask images read in use positive
+# values to indicated masked areas instead of non-masked areas.
+invert_masks = false
+
+# If true, extract metadata from the video and use metadata to
+# guide which frames are tracked.
+use_video_metadata = true
+
+# A majority of the time, mask images are a single channel,
+# however it is feasibly possible that certain
+# implementations may use multi-channel masks. If this is
+# true we will expect multiple-channel mask images,
+# warning when a single-channel mask is provided. If this
+# is false we error upon seeing a multi-channel mask image.
+expect_multichannel_masks = false
+
+# select the descriptor type
+descriptor_type := ocv_ORB
+descriptor_binary := true
+
+# Parameters for the feature tracker
+block feature_tracker
+  include core_feature_tracker.conf
+
+  # The maximum number of frames for the GUI to process.
+  # The tracker will choose frames distributed over the video
+  max_frames = 500
+endblock
```

Comment:
I just checked and it looks like you only need to override values you want to change from the default configuration.  So this entire file could be
```suggestion
# Only process a subset of frames for testing
feature_tracker:max_frames = 100
```